src/components/dialogs/create-case-dialog.tsx

```diff
@@ -48,16 +43,28 @@ const CreateCaseDialog: React.FunctionComponent<ICreateCaseDialogProps> = ({
     const dispatch = useDispatch();
     const { snackError } = useSnackMessage();
 
+    const schema = yup.object().shape({
+        [CASE_NAME]: yup
+            .string()
+            .test(
```

Comment:
Why not just `.required` ? It's what it's for, we already do that everywhere we need a string not to be empty
https://github.com/jquense/yup#stringrequiredmessage-string--function-schema

Reply:
done, thank you